package org.chainsys.myexception;

public class ExistingMailIdException extends Exception{

	{
		System.out.println("invalid mailId !! this mail id already existing");
	}
}
